app/controllers/catalog_controller.rb

```diff
@@ -2006,7 +2006,8 @@ def replace_right_cell(options = {})
 
     allowed_records = %w(MiqTemplate OrchestrationTemplate Service ServiceTemplate ServiceTemplateCatalog)
     record_showing = (type && allowed_records.include?(TreeBuilder.get_model_for_prefix(type)) && !@view) ||
-                     params[:action] == "x_show"
+                     params[:action] == "x_show" ||
+                     (%w(accordion_select tree_select).include?(params[:action]) && @record.present? && type == 'st')
```

Comment:
Looks like we're not doing that `&& !@view` anywhere else for `record_showing` (or `leaf_record`).

So, maybe that's the bug?

But, I'm inclined to merge as is, because adding extra specific cases can probably break less code :).